Source/DafnyRuntime/DafnyRuntimeGo/DafnyRuntimeFromDafny.go

```diff
@@ -309,7 +309,8 @@ func (_static *CompanionStruct_Sequence_) Create(cardinality uint32, initFn func
 func (_static *CompanionStruct_Sequence_) EqualUpTo(left Sequence, right Sequence, index uint32) bool {
   var ret bool = false
   _ = ret
-  var _hi0 = index
+  var _hi0 uint32 = index
+  _ = _hi0
```

Comment:
The Go backend emits dummy reads to ensure that the variable is used. This actually just brings the local variable declaration in line with the others.